client/src/components/bonuses.jsx

```diff
@@ -13,24 +12,34 @@ class Bonuses extends React.Component {
       selectedSong: '', //string, representing url
     };
 
+    this.selectAlbum = this.selectAlbum.bind(this);
     this.selectSong = this.selectSong.bind(this);
   }
 
   componentDidMount() {
-    let bundleId = parseInt(window.location.pathname.slice(6));
-    console.log(bundleId)
+    let bundleId = window.location.pathname;
+
+    if (bundleId === '/') {
+      bundleId = 1;
```

Comment:
So to be clear, the way you're handling this, if a user goes to the root page, i.e., `'/'`, they'll remain on the root page rather than being pathed along to `'/1'` and just the data from `'/1'` will be displayed on `'/'`?

Reply:
For now yes. I am just setting 1 as default at the   `/` directory. There may be a separate occasion where the URL will actually be `/1` so I am choosing to handle those two events as different cases though they point to the same bundle.